src/components/cards/featured-card.jsx

```diff
@@ -0,0 +1,25 @@
+export default function FeaturedCard(props) {
```

Comment:
My suggestion is to use named exports instead of default exports. Same here as well you can keep all in one file and you can import them in a single line. Once refer below file you will get an idea.

https://github.com/SRUJANA-PENUGONDA13/Local-Tailor-V1/blob/develop/src/frontend/pages/index.js